api_signaling.go

```diff
@@ -325,6 +325,7 @@ const (
 	// Features for all clients.
 	ServerFeatureMcu                   = "mcu"
 	ServerFeatureSimulcast             = "simulcast"
+	ServerFeaturePublisherUpdate      = "publisher-update"
```

Comment:
```suggestion
	ServerFeaturePublisherUpdate       = "publisher-update"
```

This should have been caught by the CI linter.

Reply:
Linter is updated in #196